### DISCUSSION TITLE
SakuraFrp 启动器安装指南Linux桌面：第3步的疑问

### DISCUSSION THREAD

**Discussion by RaspKongJu** [natfrp/wiki#436] [使用问题] [answered]
![屏幕截图 2024-02-06 201239](https://github.com/natfrp/wiki/assets/159135493/018d960c-cac4-485f-a7b6-aba179de3630)
配置service脚本时遇到（code=exited, status=203/EXEC）的报错，上网之后在开头加了#!/bin/sh之后仍未解决，权限有给，是缺了什么吗？
能启动服务，但是查询状态时就是会报错，也不能正常使用

**Comment by fakeboboliu** [comment]
203 表示找不到文件或无法执行。
在这个特定上下文中，我算卦认为是因为文件放错了位置，或者下载了错误架构的文件
（特别的：`amd64`≠`arm64`）

**Comment by RaspKongJu** [comment]
啊啊，我架构改过来了，但还是报203错误
![屏幕截图 2024-02-06 214141](https://github.com/natfrp/wiki/assets/159135493/c8cc7a28-f2c0-450c-828c-ae2b4d6c1912)

![屏幕截图 2024-02-06 214232](https://github.com/natfrp/wiki/assets/159135493/06bbee3d-0481-4a35-bba9-64e48faa7ede)

**Reply by fakeboboliu** [reply, marked as answer]
![image](https://github.com/natfrp/wiki/assets/7552030/405f1342-8f30-4ff1-9596-8ca81688817b)

![image](https://github.com/natfrp/wiki/assets/7552030/287ecb5f-73d3-47af-883d-ef08aa2b9805)

请允许我在这里放两张图

**Reply by RaspKongJu** [reply]
主要是我的树莓派没有.config目录，所以才改到/etc/systemd/system目录下，service文件里也已经改过来了，所以会有问题吗

**Reply by fakeboboliu** [reply]
> 所以会有问题吗

您好，有的

虽然不按教程不一定不工作，但如果不是熟练用户，不按教程多半不工作

**Reply by RaspKongJu** [reply]
好吧，谢谢你的答复

**Reply by fakeboboliu** [reply]
如果您坚持希望用 system systemd 的话，建议您参考 桌面教程 隔壁的 服务器教程。

这两个教程主要的对象以及最终效果都在教程最开始了，
如果您希望的话，也可以参考这两个教程自行混搭操作。

**Reply by RaspKongJu** [reply]
好的

**Reply by MarchettiJ** [reply]
您好，我使用的是linux服务器，我把配置文件放在/etc/systemd/system目录下，我也是出现了（code=exited, status=203/EXEC）的报错，natfrp.service单独运行的时候配置文件是可以正常生成的，但是无法使用/etc/systemd/system/natfrp.service来启动软件，请问是有那些问题吗？以下是我的配置截图：
![微信图片_20240314145549](https://github.com/natfrp/wiki/assets/144435375/5380de12-2b34-4a76-8985-bb70500f8e1f)

![微信图片_20240314145613](https://github.com/natfrp/wiki/assets/144435375/8658e84d-a350-4398-95e3-1ce960615536)


期待您的答复，万分感谢

[a comment by an automated account is not shown]
